src/main/java/se/sundsvall/supportmanagement/integration/db/model/StatusTagEntity.java

```diff
@@ -0,0 +1,172 @@
+import static javax.persistence.GenerationType.IDENTITY;
+
+import java.io.Serializable;
+import java.time.OffsetDateTime;
+import java.util.Objects;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+import javax.persistence.Index;
+import javax.persistence.PrePersist;
+import javax.persistence.PreUpdate;
+import javax.persistence.Table;
+import javax.persistence.UniqueConstraint;
+
+@Entity
+@Table(name = "status_tag", indexes = {
+	@Index(name = "idx_namespace_municipality_id", columnList = "namespace, municipality_id")
+}, uniqueConstraints = {
+	@UniqueConstraint(name = "uq_namespace_municipality_id_name", columnNames = { "namespace", "municipality_id", "name" })
+})
+public class StatusTagEntity implements Serializable {
+
+	private static final long serialVersionUID = -5979976910282343331L;
+
+	@Id
+	@GeneratedValue(strategy = IDENTITY)
+	@Column(name = "id")
+	private Long id;
+
+	@Column(name = "name", nullable = false)
+	private String name;
+
+	@Column(name = "municipality_id", nullable = false)
+	private String municipalityId;
+
+	@Column(name = "namespace", nullable = false)
+	private String namespace;
+
+	@Column(name = "created")
+	private OffsetDateTime created;
+
+	@Column(name = "modified")
+	private OffsetDateTime modified;
+
+	public static StatusTagEntity create() {
+		return new StatusTagEntity();
+	}
+
+	public Long getId() {
+		return id;
+	}
+
+	public void setId(Long id) {
+		this.id = id;
+	}
+
+	public StatusTagEntity withId(Long id) {
+		this.id = id;
+		return this;
+	}
+
+	public String getName() {
+		return name;
+	}
+
+	public void setName(String name) {
+		this.name = name;
+	}
+
+	public StatusTagEntity withName(String name) {
+		this.name = name;
+		return this;
+	}
+
+	public String getMunicipalityId() {
+		return municipalityId;
+	}
+
+	public void setMunicipalityId(String municipalityId) {
+		this.municipalityId = municipalityId;
+	}
+
+	public StatusTagEntity withMunicipalityId(String municipalityId) {
+		this.municipalityId = municipalityId;
+		return this;
+	}
+
+	public String getNamespace() {
+		return namespace;
+	}
+
+	public void setNamespace(String namespace) {
+		this.namespace = namespace;
+	}
+
+	public StatusTagEntity withNamespace(String namespace) {
+		this.namespace = namespace;
+		return this;
+	}
+
+	public OffsetDateTime getCreated() {
+		return created;
+	}
+
+	public void setCreated(OffsetDateTime created) {
+		this.created = created;
+	}
+
+	public StatusTagEntity withCreated(OffsetDateTime created) {
+		this.created = created;
+		return this;
+	}
+
+	public OffsetDateTime getModified() {
+		return modified;
+	}
+
+	public void setModified(OffsetDateTime modified) {
+		this.modified = modified;
+	}
+
+	public StatusTagEntity withModified(OffsetDateTime modified) {
+		this.modified = modified;
+		return this;
+	}
+
+	@PrePersist
+	void onCreate() {
```

Comment:
Why is the visibility "package private" on this method, but not on onUpdate()?

Reply:
Good question, this seem to be implemented on other entities as well. Will change protected to default visibility for all prePresist/preUpdate methods.